INFO:root:RPC client connected - server_port=1234, server_ip='192.168.40.23', ssl=True
INFO:root:Console created - self.cid='0'
INFO:root:session created - 1
INFO:root:using payload - cmd/unix/bind_netcat
INFO:root:using exploit - multi/samba/usermap_script
INFO:root:unable to create session
INFO:root:session created - 2
INFO:root:Meterpreter shell session created - diff={'3'}
INFO:root:unable to create session
WARNING:retry.api:HTTPSConnectionPool(host='192.168.40.23', port=1234): Max retries exceeded with url: /api/ (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f648ca80f10>: Failed to establish a new connection: [Errno 111] Connection refused')), retrying in 1 seconds...
WARNING:retry.api:HTTPSConnectionPool(host='192.168.40.23', port=1234): Max retries exceeded with url: /api/ (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f648ca70b80>: Failed to establish a new connection: [Errno 111] Connection refused')), retrying in 2 seconds...
